docs/core/docker/build-container.md

```diff
@@ -0,0 +1,391 @@
+04/05/2019  11:00 AM               448 myapp.pdb
+04/05/2019  11:00 AM               154 myapp.runtimeconfig.json
+```
+
+```bash
+me@DESKTOP:/path-to-working-dir/app$ ls bin/Release/netcoreapp2.2/publish
+myapp.deps.json  myapp.dll  myapp.pdb  myapp.runtimeconfig.json
+```
+
+In your terminal, go back a directory to the working directory.
+
+## Create the Dockerfile
+
+The *Dockerfile* file is used by the `docker build` command to create a container image. This file is a plaintext file named *Dockerfile* that does not have an extension. Create a file named *Dockerfile* in your working directory and open it in a text editor. Add the following command as the first line of the file:
+
+```dockerfile
+FROM mcr.microsoft.com/dotnet/core/runtime:2.2
+```
+
+The `FROM` command tells Docker to pull down the image tagged **2.2** from the **mcr.microsoft.com/dotnet/core/runtime** repository. Make sure that you pull the .NET Core runtime that matches the runtime targeted by your SDK. For example, the app created in the previous section used the .NET Core 2.2 SDK and created an app that targeted .NET Core 2.2. So the base image referred to in the *Dockerfile* is tagged with **2.2**.
+
+Save the file. From your terminal, run `docker build -t myimage .` and Docker will process each line in the *Dockerfile*. The `.` in the `docker build` command tells docker to use the current directory to find a *Dockerfile*. This command builds the image and creates a local repository named **myimage** that points to that image. After this command finishes, run `docker images` to see a list of images installed:
+
+```console
+> docker images
+REPOSITORY                              TAG                 IMAGE ID            CREATED             SIZE
+mcr.microsoft.com/dotnet/core/runtime   2.2                 d51bb4452469        2 days ago          314MB
+myimage                                 latest              d51bb4452469        2 days ago          314MB
+```
+
+Notice that the two images share the same **IMAGE ID** value. The value is the same between both images because the only command in the *Dockerfile* was to base the new image on an existing image. Comparing the two images, they're exactly the same. Let's add two commands to the *Dockerfile*. Each command will create a new image layer with the final command representing the image the **myimage** repository will point to.
+
+
+```dockerfile
+COPY app\bin\Release\netcoreapp2.2\publish\ .\app
+
+ENTRYPOINT ["dotnet" "app/myapp.dll"]
+```
+
+The `COPY` command tells Docker to copy the specified folder on your computer to a folder in the container. In this example, the **publish** folder is copied to a folder named **app** in the container.
+
+The next command, `ENTRYPOINT`, tells docker to configure the container to run as an executable. After you start a container based on this image, it will automatically stop when the process identified from the `ENTRYPOINT` command ends.
```

Comment:
You say the container stops when the program ends. I'd add a sentence before that (or modify the first sentence) to say that the container starts running the program identified by `ENTRYPOINT`.